package.json

```diff
@@ -33,43 +33,43 @@
   },
   "pre-commit": [],
   "devDependencies": {
-    "assets-webpack-plugin": "git://github.com/mattkrick/assets-webpack-plugin.git/#master",
+    "assets-webpack-plugin": "https://github.com/mattkrick/assets-webpack-plugin/tarball/master",
```

Comment:
i :heart: the `/tarball/master` deal. plus, the `git` wasn't giving me all the files, which was weird.